ffi_derive/src/lib.rs

```diff
@@ -200,6 +200,39 @@
 //! `pub field foo: crate1::aliases::Foo`), we won't be able to figure out how to go from that
 //! definition to `Foo` to `u8`.
 //!
+//! ## Deriving on an impl
+//!
+//! We also support generating an FFI for a trait implemented on a type with the `expose_items`
+//! attribute macro.
+//!
+//! The only caveat here is that the invocation site needs to provide the paths to the FFI modules
+//! that we'll need to import. For example, the FFI form of a function like
+//! `fn meow(&self, volume: Option<Volume>) -> Vec<Meow> { ... }` will need to know the paths to the
+//! FFI types of `Meow` and `Volume`. Fortunately, since those FFI types probably come from
+//! `ffi_derive`, it's relatively easy to figure out what they would be based on your normal
+//! imports. If you do something like `use crate::animals::cats::Meow;` and
+//! `use utilities::sound::Volume;`, you'll just need to provide
+//! `"crate::animals::cats::meow_ffi", "utilities::sound::volume_ffi"` to the attribute macro.
```

Comment:
Dunno if this is quite relevant, but maybe worth throwing out there... with declarative macros (aka normal macros), when referencing types or functions from another module or crate, it's conventional to use the full path to thing. Requiring uses to do the same here wouldn't be a stretch IMO.

Reply:
That makes sense, and is probably safer 👍  It feels a little weird to need to know the path of something that you can't see without doing `cargo expand` (or letting rust-analyzer do it for you), but it shouldn't be too weird if it's normal to have to provide the full path for stuff like this.